src/graphics/graphics.c

```diff
@@ -24,22 +22,13 @@ static struct {
 static struct {
     int x;
     int y;
-} translation = {0, 0};
+} translation;
 
 static clip_info clip;
 
-#ifdef __vita__
```

Comment:
Nice! No more platform checks in the main Julius code.

Reply:
Yes, precisely one of the reasons I've been wanting to do this for a long time...